### PR TITLE
JeniusApps.Ambie version 2.0.14.0

### PR DESCRIPTION
**Pull request by zephiowl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/33294)

### CODE REVIEW

**Comment by Xanthus58** [comment]
image


This definitely works locally; I'm going to assume that the install file (Ambie_2.0.14.0_x86_x64_ARM64.msixbundle) Includes/Requires some dependency's that are not currently present on the testing bench. Causing this error, its my best guess. 

If i could grab some help that'd be great :)

File: manifests/j/JeniusApps/Ambie/2.0.14.0/JeniusApps.Ambie.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+PackageName: Ambie
+PackageUrl: https://github.com/jenius-apps/ambie
+License: MIT
+LicenseUrl: https://github.com/jenius-apps/ambie/blob/main/LICENSE.txt
+Copyright: https://github.com/jenius-apps/ambie/blob/main/LICENSE.txt
+ShortDescription: The best white noise app on Windows
+Description: Ambie is a mindful focused application that plays ambient sounds and create mixes to help you focus, sleep, or meditate.
+Moniker: Ambie
+Tags:
+- Health & Fitness
```
Comment (ItzLevvie) [inline manifests/j/JeniusApps/Ambie/2.0.14.0/JeniusApps.Ambie.locale.en-US.yaml:20]:
```suggestion
- health
- fitness
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I don't have an arm64 device to test - only x64 and x86 - but I'm going to guess that x86 or arm64 installer (or both) has missing dependencies which should be reported to the developer, so they can fix it.

If you only include the x64 installer in the package for now, I believe it should pass.

File: manifests/j/JeniusApps/Ambie/2.0.14.0/JeniusApps.Ambie.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  PackageFamilyName: 43891JeniusApps.Ambie_jaj7tphbgjeh8
+- Platform:
+  - Windows.Universal
+  MinimumOSVersion: 10.0.17763.0
+  Architecture: arm64
+  InstallerType: msix
+  InstallerUrl: https://github.com/jenius-apps/ambie/releases/download/v2.0.14/Ambie_2.0.14.0_x86_x64_ARM64.msixbundle
+  InstallerSha256: 41175AC8CB5CD46992A57A0C849D8F6BE4AB8BA1489DB7D278E71291656A2328
+  SignatureSha256: A22FED30D01DE18528D27557643C115D486C4D72AF9A243F98C805C03CBD0DD1
+  PackageFamilyName: 43891JeniusApps.Ambie_jaj7tphbgjeh8
```
Comment (ItzLevvie) [inline manifests/j/JeniusApps/Ambie/2.0.14.0/JeniusApps.Ambie.installer.yaml:33]:
```suggestion
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by Xanthus58** [comment]
Well; that didn't work

**Comment by ItzLevvie** [comment]
I'll take a look ...

Ah it requires these dependencies:

```
<PackageDependency Name="Microsoft.UI.Xaml.2.5" MinVersion="2.52012.2002.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />
<PackageDependency Name="Microsoft.NET.Native.Framework.2.2" MinVersion="2.2.29512.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />
<PackageDependency Name="Microsoft.NET.Native.Runtime.2.2" MinVersion="2.2.28604.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />
<PackageDependency Name="Microsoft.VCLibs.140.00" MinVersion="14.0.30035.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />
<PackageDependency Name="Microsoft.Services.Store.Engagement" MinVersion="10.0.19011.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />
```

`Microsoft.UI.Xaml.2.5` is a dependency that's probably not included in the pipelines (and probably other dependencies included), so this package might not be able to pass.

**Comment by Xanthus58** [comment]
> `Microsoft.UI.Xaml.2.5` is a dependency that's probably not included in the pipelines (and probably other dependencies as well), so this package might not be able to pass right now.

Ah i see, so i should close it and reopen it later down the line?

**Comment by ItzLevvie** [comment]
Since the pipelines are running on Windows 10, I believe the pipelines has MUX 2.4 and below. It probably takes a lot of effort to update the pipelines with the latest version of Windows.

MUX 2.5 seems to be an uncommon/rare dependency in some apps, I mostly only see apps from https://uwpcommunity.com/projects use MUX 2.5.
MUX 2.6 and MUX 2.7 are commonly seen in Windows 11.

>Ah i see, so i should close it and reopen it later down the line?

At the moment, WinGet currently doesn't have full dependency support (i.e. external dependency for MUX - because if that happened I could contact the team to create a URL for it, so it could be included in the package) right now, only package dependencies (a package installs another package in the repo) which reached in winget-cli's master branch.

So, you could probably take a look at it in the future.

**Pull request closed without merging**